tcl/dialog_audio.tcl

```diff
@@ -202,6 +203,17 @@ proc ::dialog_audio::pdtk_audio_dialog {mytoplevel \
     pack $mytoplevel.inputs.in1f.x1 -side left -fill x -expand 1
     pack $mytoplevel.inputs.in1f.x2 $mytoplevel.inputs.in1f.l2 -side right
 
+    # FIXME: simple hack to disable samplerate when using JACK
```

Comment:
I find it useful to display the current sr used by JACK, partially as a confirmation that it matches that displayed in something like qjackctl. However, another approach would be to simply hide all of the sr guis if the sr is fixed. IN this case, as you suggest, a value of `0` would indicate sr is fixed and no GUI is required.

Reply:
I like the idea of a negative number to indicate something, but I'm unsure if they could lead to blowups if either side (GUI or core) misinterpret the number...

Reply:
i kind of like the `!` prefix as it is *obviously* an invalid number - so if something goes wrong it will blow-up sooner rather than later, making it easy to spot.

it would also allow to use just `!` as an indicator for *no-display* while still allowing the `0` value.